WorkflowUI/Sources/Hosting/WorkflowHostingController.swift

```diff
@@ -22,7 +22,7 @@ import UIKit
 import Workflow
 
 /// Drives view controllers from a root Workflow.
-public final class WorkflowHostingController<ScreenType, Output>: UIViewController where ScreenType: Screen {
+public final class WorkflowHostingController<ScreenType, Output>: WorkflowUIViewController where ScreenType: Screen {
```

Comment:
@kyleve responding here for threading purposes:

> >ViewControllerDescriptions not describing a ScreenViewController subclass does seem like a potential footgun here
> 
> Agreed. I don't think this is something we can/should enforce here, IMO.

could you clarify – you think this is a risk, but not one we should change the contract of ViewControllerDescription to deal with?